src/gui/group/GroupView.cpp

```diff
@@ -60,6 +62,26 @@ GroupView::GroupView(Database* db, QWidget* parent)
     setDefaultDropAction(Qt::MoveAction);
 }
 
+void GroupView::prevGroup()
+{
+    auto prevIndex = indexAbove(currentIndex());
+    if (!prevIndex.isValid())
+        return;
```

Comment:
Our code style requires braces even for single line if statements. For this function, I recommend doing this instead:

```
if (prevIndex.isValid()) {
    auto prevGroup = m_model->groupFromIndex(prevIndex);
    setCurrentGroup(prevGroup);
}
```